gnd/src/main/java/com/google/android/gnd/repository/UserRepository.java

```diff
@@ -29,15 +30,22 @@
 public class UserRepository {
 
   private final LocalDataStore localDataStore;
+  private final LocalValueStore localValueStore;
   private final Schedulers schedulers;
 
   @Inject
-  UserRepository(LocalDataStore localDataStore, Schedulers schedulers) {
+  UserRepository(LocalDataStore localDataStore, LocalValueStore localValueStore, Schedulers schedulers) {
```

Comment:
Checkstyle error:
 /workspace/gnd/src/main/java/com/google/android/gnd/repository/UserRepository.java:37: Line is longer than 100 characters (found 105). [LineLength]